ci/pipeline.yml

```diff
@@ -238,6 +238,7 @@ jobs:
             bosh-agent: bumped-bosh-agent
           tags:
             - windows-nimbus
+            - windows-2019
```

Comment:
I think you want to remove the `windows-nimbus` tag if you only want it to run on the `2019` worker

Reply:
We believe that tags are ANDed, so we’d only ever pick workers with both tags. Does that not match your understanding?

Reply:
Or you just think the windows-nimbus tag is superfluous?

Reply:
> Or you just think the windows-nimbus tag is superfluous?

yup

Reply:
Done.